go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
 	github.com/bketelsen/buffet v0.1.5
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
+	github.com/codegangsta/negroni v0.3.0 // indirect
```

Comment:
I did a global search on this import path (and others) and they seem to be not imported at all by any of the vendor directory. You probably ran `go mod tidy` and ended up with something like this issue: https://github.com/golang/go/issues/27428

Reply:
I do not think it was added incorrectly, it is indeed an indirect dep. 

`go mod why -m github.com/codegangsta/negroni`
```bash
# github.com/codegangsta/negroni
github.com/gomods/athens/cmd/olympus/actions
github.com/unrolled/secure
github.com/unrolled/secure.test
github.com/codegangsta/negroni
```

Reply:
@komuw tests of dependencies are not included in vendor, nor are they tested when you say `go test ./...` -- but in any case. This is not biggie, just something to be aware of :) thank you for fixing the PR!